pyvista/core/filters/poly_data.py

```diff
@@ -2756,6 +2767,15 @@ def extrude_rotate(
         alg.SetDeltaRadius(dradius)
         alg.SetCapping(capping)
         alg.SetAngle(angle)
+        if hasattr(alg, 'SetRotationAxis'):
```

Comment:
Do we have a consistent way we handle these type of functionality checks that are related to versions in vtk?  Now that we have a builtin way to access the vtk version, I think it is cleaner to use the explicit version of vtk.  Using the `hasattr` method, if vtk changes the name of this method, we won't catch it in our tests.

I don't insist on this, just my opinion.

Reply:
> if vtk changes the name of this method, we won't catch it in our tests.
I did not notice it. It would certainly be more reliable to specify the version. I was referring to the way in the transform method so we better fix that in other PR too.
```suggestion
        if pyvista.vtk_version_info >= (9, 1, 0):
```